.env.example

```diff
@@ -5,7 +5,7 @@ MQTT_PW=TODO
 
 POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
-POSTGRES_DB=thingy-db-red
+POSTGRES_MULTIPLE_DATABASES="thingy-db-red","thingy-db-red-test"
```

Comment:
@pf15ese why introducing another database? I think we could simply use another scheme or startup a docker service without any volume mount to avoid persist the dummy-data.